### ISSUE TITLE
Wrong compilation when a method has an argument with the same name than an import

### ISSUE THREAD

**Issue by dpinart-uxland** [fuse-box/fuse-box#1921]
I think I found a funky error in the compiler. I have a project that uses Polymer that has as dependency the @webcomponents project. One of the webcomponents libs, shadycss, has a module, style-util.js that contains the following import:
```
import {nativeShadow, nativeCssVariables, **cssBuild**} from './style-settings.js';

```
As you can see, it's importing the **cssBuild** property from './style-settings.js;

At the same time, style-util.js has the _isOptimalCssBuild_ function

```
/**
 * Check if the css build status is optimal, and do no unneeded work.
 *
 * @param {string=} cssBuild CSS build status
 * @return {boolean} css build is optimal or not
 */
export function isOptimalCssBuild(cssBuild = '') {
  // CSS custom property shim always requires work
  if (cssBuild === '' || !nativeCssVariables) {
    return false;
  }
  return nativeShadow ? cssBuild === 'shadow' : cssBuild === 'shady';
}
```
As you can see, _isOptimaCssBuild_ function has an argument named **cssBuild** . The fact is that fuse-box is wrongly compiling the _isOptimalCssBuild_. The generated javascript is as follows

```
function isOptimalCssBuild(style_settings_js_1.cssBuild = '') {
  if (style_settings_js_1.cssBuild === '' || !style_settings_js_1.nativeCssVariables) {
    return false;
  }
  return style_settings_js_1.nativeShadow ? style_settings_js_1.cssBuild === 'shadow' : style_settings_js_1.cssBuild === 'shady';
}
exports.isOptimalCssBuild = isOptimalCssBuild;
```
Instead of respecting the argument name, is replacing it by the import

**Comment by nchanged** [comment]
@dpinart-uxland thanks! that's a confirmed bug! I will fix it soon

**Comment by nchanged** [comment]
Sorry it took a while, this one is fixed in `4.0.0-next.436`

**Comment by dpinart-uxland** [comment]
Sorry but I'm afraid the fix introduced a new bug. 4.0.0-next.436. Lit-element lib has a module named _updating_element_. This module exports a const as
```
/**
 * Change function that returns true if `value` is different from `oldValue`.
 * This method is used as the default for a property's `hasChanged` function.
 */
export const notEqual = (value, old) => {
    // This ensures (old==NaN, value==NaN) always returns false
    return old !== value && (old === old || value === value);
};

```
This module also exports a large class _UpdatingElement_ that is used as the base for creating cusomt web components. This class has the following declaration

```
 /**
     * Returns true if a property should request an update.
     * Called when a property value is set and uses the `hasChanged`
     * option for the property if present or a strict identity check.
     * @nocollapse
     */
    static _valueHasChanged(value, old, hasChanged = notEqual) {
        return hasChanged(value, old);
    }
```
__valueHasChanged_ receives 3 arguments, the 2 first are values and the last one is the function used to compare the values. The third arg is defaulted to the previous exported function.

Compiling with version 436 I get an error when executing the app:
**ReferenceError: notEqual is not defined**

Fusebox is compiling this file as:
```
exports.notEqual = (value, old) => {
  return old !== value && (old === old || value === value);
};
....

static _valueHasChanged(value, old, hasChanged = notEqual) {
    return hasChanged(value, old);
  }
```

I susspect that's becasue fuse-box is now defining _notEqual_ direclty in exports, so current module cannot reference it internally

**Comment by nchanged** [comment]
@dpinart-uxland please verify the fix in `4.0.0-next.438`

**Comment by dpinart-uxland** [comment]
Yes.  It now compiles fine and app starts. I'm seeing some errors in console but not sure if they are compilation errors or app bugs. I'll open an issue if I finally detect they are compilation errors
